### PR TITLE
chore: upgrade requirements

### PR DESCRIPTION
**Pull request by alangsto**
**Description:** The docker publish step is failing with  `ERROR: Failed building wheel for <package name>`. It looks like this may be an error with how `wheel` is being installed. I noticed that the version of `wheel` defined by `pip.txt` differs from other requirements files. Trying to upgrade the package to see if this helps. 

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)

### CODE REVIEW

**Review by schenedx — APPROVED** [review]
👍

**Comment by m-pitera** [comment]
### Documenting some Wheel notes:

Seems like Wheel just “makes things go fast” in terms of installing, but defaults to source distro if failed.

TLDR from this [quick read](https://realpython.com/python-wheels/#wheels-make-things-go-fast): if a wheel and source distribution is offered pip will prefer the wheel, because it’s smaller and faster.
TLDR from this [stackOverflow Q](https://stackoverflow.com/questions/53204916/what-is-the-meaning-of-failed-building-wheel-for-x-in-pip-install): if package not a wheel, pip will try to build one for it. If this fails it’ll try to install through setup.py install. `pip install wheel` is referenced as a potential solution if wheel fails to build
